configuration/configuration.py

```diff
@@ -119,6 +119,10 @@ def read_secret(secret_name):
     'PASSWORD': os.environ.get('EMAIL_PASSWORD', read_secret('email_password')),
     'TIMEOUT': int(os.environ.get('EMAIL_TIMEOUT', 10)),  # seconds
     'FROM_EMAIL': os.environ.get('EMAIL_FROM', ''),
+    'USE_SSL': os.environ.get('EMAIL_USE_SSL', 'False'),
+    'USE_TLS': os.environ.get('EMAIL_USE_TLS', 'False'),
```

Comment:
You're settings `USE_SSL` and `USE_TLS` to a string, but it should be a boolean. Have a look at how it is done with `METRICS_ENABLED`.

Reply:
Implemented. Thanks for the advice.